resources/web/wwi/nodes/WbCadShape.js

```diff
@@ -0,0 +1,75 @@
+import WbBaseNode from './WbBaseNode.js';
+import WbWorld from './WbWorld.js';
+import {getAnId} from './utils/utils.js';
+
+export default class WbCadShape extends WbBaseNode {
```

Comment:
The reason was that once webotsjs will support tessellation, it will be closer to a WbBaseNode than a WbGroup, but you're right, now it's definitely the opposite